opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -1331,7 +1331,12 @@ type Member {
   id: ID! # internal_id
   name: String!
   entity_type: String!
+  # entity_type == User
+  effective_confidence_level: EffectiveConfidenceLevel
+  # entity_type == Group
+  group_confidence_level: ConfidenceLevel
```

Comment:
RFC here

Reply:
where is the resolver for this field ? I see only effective_confidence_level resolver for Member.

Reply:
this field does not need specific resolution. We pull the entities from elastic and `group_confidence_level` is one of the field returned in case it's a group.

Reply:
do we need it somewhere ? I can't find where we use it in the frontend calling members API.

Reply:
indeed, `group_confidence_level` is unused for now but I added in anticipation.